lib/app/views/test_list.erb

```diff
@@ -46,6 +46,11 @@
             <span class="oi oi-warning"></span>
           </div>
       <% end %>
+      <% if result.validation_messages.length > 0 %>
+          <div class="result-details-icon result-details-icon-info" style="float:right;" data-toggle="tooltip" title="<%= result.validation_messages.length %> informational message(s). Messages do not result in a test failure.">
+            <span class="oi oi-info"></span>
```

Comment:
The icon isn't appearing for me here:
![Screen Shot 2020-01-07 at 8 58 00 AM](https://user-images.githubusercontent.com/15969967/71900471-e1362c00-312b-11ea-86ea-b7f7f34204de.png)

Reply:
I think it's because I had to add a color to `styles.css` in order to get the icon to show up. Hard reload to clear the old css out of the cache should do it.